opensafely/__init__.py

```diff
@@ -42,6 +43,7 @@ def add_subcommand(cmd, module):
     add_subcommand("check", check)
     add_subcommand("jupyter", jupyter)
     add_subcommand("unzip", unzip)
+    add_subcommand("log-stats", log_stats)
```

Comment:
If I was going to be _super_ bikesheddy I might say the main verb should be something like "extract" here rather than "log" (e.g. `extract-stats` or `extract-logs`) because that's a more accurate description of what it's doing. I don't hugely care about this, but given that it's awkward to change it later it's maybe worth ~3 minutes of thought.

Reply:
yes, that's better. I wasn't happy with the command nane anyway